Formula/gcc.rb

```diff
@@ -101,11 +90,8 @@ def install
     args << "--enable-libphobos" if Hardware::CPU.intel?
 
     if OS.mac?
+      cpu = Hardware::CPU.arm? ? "aarch64" : "x86_64"
       args << "--build=#{cpu}-apple-darwin#{OS.kernel_version.major}"
-      args << "--with-system-zlib"
```

Comment:
Does this use a vendored zlib then?

Reply:
No it's just the default (has always been, as far as I can remember).

Reply:
Actually, after double-checking, that was wrong of me, and I've restored the configure flag.